nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-standard-record-utils/src/main/java/org/apache/nifi/record/listen/SocketChannelRecordReaderDispatcher.java

```diff
@@ -94,7 +95,7 @@ public void run() {
 
                 if (logger.isDebugEnabled()) {
                     final String remoteAddress = remoteSocketAddress == null ? "null" : remoteSocketAddress.toString();
-                    logger.debug("Accepted connection from {}", new Object[]{remoteAddress});
+                    logger.debug("Accepted connection from {}", remoteAddress);
```

Comment:
Although this change is helpful, recommend reverting it from this PR since it is unrelated.